README.md

```diff
@@ -105,6 +105,30 @@ logger = logging.getLogger(__name__)
 logger.info('testing123...')
 ```
 
+## Clients
+Client connects with server.py on host machine to communicate with modules. Here is an example of creating a client.
+```python
+from client import client
+client_1 = client()
+```
+
+ping(self) can be issued to ping the server to get the client's IP address and binding port.
+```python
+print(client_1.ping())
+```
+
+com(self, module, funcname, *args) can be called to communicate with a module and request it to perform a function. Here is an example of requesting "moduleB" to call "fun1" with arguments ['ay', 1, False, None].
+```python
+resp = client_1.com('moduleB', 'fun1', 'ay', 1, False, None)
```

Comment:
Is this the `moduleB` from above? If so, can you also print the expected output/contents of `resp` so users can compare.